lib/core/elastic_connector_actions.rb

```diff
@@ -41,14 +41,17 @@ def self.load_connector_settings(connector_package_id)
     end
 
     def self.update_connector_configuration(connector_package_id, configuration)
-      body = {
-        :doc => {
-          :configuration => configuration
-        }
-      }
+      update_connector_data(connector_package_id, :configuration, configuration)
+    end
 
-      client.update(:index => CONNECTORS_INDEX, :id => connector_package_id, :body => body)
-      Utility::Logger.info("Successfully updated configuration for connector #{connector_package_id}")
+    def self.enable_connector_scheduling(connector_package_id, cron_expression)
```

Comment:
Good catch. If you're switching it on for the first time, you need a new schedule, but if it's not the first time, you might just re-enable it. I have fixed it to show the previous schedule so that it's possible to copy-paste it. It's not ideal but I think it's a workable solution.